tests/test_numpy_array.py

```diff
@@ -198,7 +198,6 @@ def assert_references(a, b, base=None):
         assert a.flags.f_contiguous == b.flags.f_contiguous
         assert a.flags.writeable == b.flags.writeable
         assert a.flags.aligned == b.flags.aligned
-        # 1.13 supported Python 3.6
         if tuple(int(x) for x in np.__version__.split(".")[:2]) >= (1, 14):
             assert a.flags.writebackifcopy == b.flags.writebackifcopy
```

Comment:
```suggestion
        assert a.flags.writebackifcopy == b.flags.writebackifcopy
```

The oldest numpy to support 3.7 was 1.14 (https://github.com/scipy/oldest-supported-numpy/blob/23ece5a161d5957f5142991393d2ae689ee45b9d/setup.cfg#L56C12-L56C18)